### PR TITLE
Euclid 4way

### PR DESCRIPTION
**Pull request by oneswig**
Some fixes from deploying to another site with slightly different configuration.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,17 @@
 ---
+
+# If we set these as facts, they are reachable via hostvars
+- set_fact:
+    openvpn_intra_if: "{{ ansible_default_ipv4.interface }}"
+  when: openvpn_intra_if is not defined
+- set_fact:
+    openvpn_intra_ipv4: "{{ hostvars[inventory_hostname]['ansible_' + openvpn_intra_if]['ipv4'] }}"
```
Comment (priteau) [inline tasks/main.yml:8]: The line above returns a list of dicts, each containing `address`, `broadcast`, `netmask`, and `network`…

File: tasks/main.yml [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+  when: openvpn_intra_ipv4 is not defined
+
+- set_fact:
+    openvpn_intra_cidr: "{{ (openvpn_intra_ipv4.network + '/' + openvpn_intra_ipv4.netmask) | ipaddr('net') }}"
```
Comment (priteau) [inline tasks/main.yml:12]: … which would make the reference to `.network` fail here.

**Review by priteau — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (priteau) [reply tasks/main.yml:8]: Actually this only happens on my macOS machine. On Linux the value of `ipv4` is always a dict, even if multiple IP addresses are configured.

File: tasks/main.yml (hunk 2 above)
Comment (priteau) [reply tasks/main.yml:12]: Nevermind, seems to be macOS-specific.

**Review by priteau — APPROVED** [review]

File: tasks/main.yml (hunk 2 above)
Comment (oneswig) [reply tasks/main.yml:12]: Thanks @priteau

**Pull request merged**